.github/workflows/push-tests.yml

```diff
@@ -312,7 +312,7 @@ jobs:
         if: steps.changed-files.outputs.any_changed == 'true'
         run: |
           echo "DOCKER_REPO=${DOCKER_REPO}" > .env.ecr
-          make local-rebuild-processing
+          make local-rebuild-wmg-processing
```

Comment:
This was a bug in the WMG processing push tests. It was rebuilding the wrong image